frontend/public/components/notification-drawer.tsx

```diff
@@ -279,6 +289,12 @@ const notificationStateToProps = ({ UI }: RootState): WithNotificationsProps =>
   alerts: UI.getIn(['monitoring', 'notificationAlerts']) || {},
 });
 
+type ActionAlertType = {
+  id: string;
+  text: string;
+  url: string;
```

Comment:
Since we use `Link` it's a relative path, not a URL

```suggestion
  path: string;
```